src/baguetteBox.scss

```diff
@@ -32,8 +32,6 @@
 
         figure {
             display: inline;
-            margin: 0;
-            height: 100%; // Opera 12 image stretching fix
```

Comment:
Height and margin have to effect with inline styles. I wonder how you say that fixed any issue.

No, I haven't tested Opera 12, but like I said, height has no effect. You might have meant inline-block, but that's your call.